Dockerfile

```
@@ -0,0 +1,24 @@
+# Set explicit platform for ARM-based Macbooks
+FROM --platform=linux/amd64 python:3.10.13-slim-bookworm
+
+# Install dependencies in container
+RUN apt-get update && \
+    apt-get install -y nodejs graphviz && \
+    pip install atoti \
+                atoti-jupyterlab \
+                atoti-aws \
+                atoti-sql
```

A little nicer with the "extras" syntax:

```suggestion
    pip install "atoti[aws,jupyterlab,sql]"
```

---

No need since we're promoting poetry installation method anyways!